sdk/eventhub/Azure.Messaging.EventHubs/tests/Amqp/AmqpMessageConverterTests.cs

```diff
@@ -710,5 +713,323 @@ public void CreateBatchFromMessagesWithMultipleEventsAssignsThePartitionKeyToBod
                 Assert.That(actual, Is.EqualTo(expected), $"The batch body for message { index } should match the serialized event.");
             }
         }
+
+        /// <summary>
+        ///   Verifies functionality of the <see cref="AmqpMessageConverter.CreateEventFromMessage" />
+        ///   method.
+        /// </summary>
+        ///
+        [Test]
+        public void CreateEventFromMessageValidatesTheSource()
+        {
+            var converter = new AmqpMessageConverter();
+            Assert.That(() => converter.CreateEventFromMessage(null), Throws.ArgumentNullException);
+        }
+
+        /// <summary>
+        ///   Verifies functionality of the <see cref="AmqpMessageConverter.CreateEventFromMessage" />
+        ///   method.
+        /// </summary>
+        ///
+        [Test]
+        public void CreateEventFromMessagePopulatesTheBody()
+        {
+            var body = new byte[] { 0x11, 0x22, 0x33 };
+
+            using var bodyStream = new MemoryStream(body, false);
+            using var message = AmqpMessage.Create(bodyStream, true);
+
+            var converter = new AmqpMessageConverter();
+            var eventData = converter.CreateEventFromMessage(message);
+
+            Assert.That(eventData, Is.Not.Null, "The event should have been created.");
+            Assert.That(eventData.Body, Is.Not.Null, "The event should have a body.");
+            Assert.That(eventData.Body.ToArray(), Is.EqualTo(body), "The body contents should match");
```

Comment:
Super nit: append period to the end of the message for consistency.

Reply:
Cool if I hit that on the PR that will immediately follow this with a bunch of little fixes that are similar?